Windows Jenkins machines were fixed, but looks like CI is failing on Windows (looks related);

```
=== RUN   TestCompile/slash
    fileutils_test.go:761: pattern "a*/b": regexp = ^a[^/]*\b$, want ^a[^/]*/b$
    fileutils_test.go:761: pattern "**/**": regexp = ^(.*/)?\*[^/]*$, want ^(.*/)?.*$
    fileutils_test.go:752: pattern "dir/**": matchType = 4, want 2
    fileutils_test.go:752: pattern "**/dir": matchType = 4, want 3
    fileutils_test.go:761: pattern "**/dir2/*": regexp = ^(.*/)?\dir2\*$, want ^(.*/)?dir2/[^/]*$
    fileutils_test.go:761: pattern "**/dir2/**": regexp = ^(.*/)?\dir2\*[^/]*$, want ^(.*/)?dir2/.*$
    fileutils_test.go:761: pattern "**/file*txt": regexp = ^(.*/)?\file[^/]*txt$, want ^(.*/)?file[^/]*txt$
    fileutils_test.go:761: pattern "**/**/*.txt": regexp = ^(.*/)?\*[^/]*\*\.txt$, want ^(.*/)?(.*/)?[^/]*\.txt$
    fileutils_test.go:752: pattern "**/foo/bar": matchType = 4, want 3
    fileutils_test.go:752: pattern "a(b)c/def": matchType = 4, want 1
    fileutils_test.go:749: Failed to compile pattern "dist/proxy.py-2.4.0rc3.dev36+g08acad9-py3-none-any.whl": error parsing regexp: invalid character class range: `\pr`
    --- FAIL: TestCompile/slash (0.00s)
```

--- a/pkg/fileutils/fileutils_test.go
+++ b/pkg/fileutils/fileutils_test.go
@@ -737,14 +737,19 @@ func TestCompile(t *testing.T) {
 func testCompile(sl string) func(*testing.T) {
 	return func(t *testing.T) {
 		for _, tt := range compileTests {
-			pattern := tt.pattern
+			// Avoid NewPatternMatcher, which has platform-specific behavior
+			pm := &PatternMatcher{
+				patterns: make([]*Pattern, 1),
+			}
+			pattern := path.Clean(tt.pattern)
 			if sl != "/" {
 				pattern = strings.ReplaceAll(pattern, "/", sl)
 			}
-			pm, err := NewPatternMatcher([]string{pattern})
-			if err != nil {
-				t.Fatalf("Failed to create PatternMatcher for pattern %q: %v", pattern, err)
-			}
+			newp := &Pattern{}
+			newp.cleanedPattern = pattern
+			newp.dirs = strings.Split(pattern, sl)
+			pm.patterns[0] = newp
+
 			if err := pm.patterns[0].compile(sl); err != nil {
 				t.Fatalf("Failed to compile pattern %q: %v", pattern, err)
 			}

